TTL_trace_macros.h

```diff
@@ -39,16 +39,13 @@
 #define TTL_step_buffering(...) TTL_step_buffering(__VA_ARGS__, __LINE__)
 
 #define TTL_start_simplex_buffering(...) TTL_start_simplex_buffering(__VA_ARGS__, __LINE__)
-#define TTL_finish_simplex_buffering(...) TTL_finish_simplex_buffering(__VA_ARGS__, __LINE__)
+#define TTL_finish_buffering(...) TTL_finish_buffering(__VA_ARGS__, __LINE__)
```

Comment:
nit: may be better to place TTL_finish_buffering next to TTL_step_buffering, rather than after TTL_start_simplex_buffering.

It might be possible to also have a single generic TTL_start_buffering, if all schemes have distinct arguments, but less informative / too obscure?